### DISCUSSION TITLE
Undocumented Numbers: delta

### DISCUSSION THREAD

**Discussion by idlebear** [TheCodez/dynamic-occupancy-grid-map#88] [Q&A] [answered]
Where does the value 0.6 come from  when delta is calculated?
```
__device__ float pOcc(int r, float zk, int index, float resolution)
{
    float occ_max = 0.95f;
    float delta = 0.6f / resolution;         // <--- this line

    return occ_max * exp(-0.5f * (index - r) * (index - r) / (delta * delta));
}
```

While this seems to work well with larger maps (100m), when I run the same code on a jackal at smaller scale (10m), the result is extra wide occupancy blocks that don't coincide with the walls.  A value of delta==1 works at a small scale, but isn't tied to the map resolution so isn't really a solution for the longer term (and looks a little thin at larger scales).

**Comment by TheCodez** [comment, marked as answer]
0.6 is a value that looked good in the demo scene. The whole pOcc should be changed to operate in meters and the `delta`(stddev) and `occ_max` parameter should then be configurable `LaserMeasurementGrid::Params`.
